src/PrestaShopBundle/Resources/views/Admin/Module/Includes/card.html.twig

```diff
@@ -49,7 +50,9 @@
             {% endif %}
           </div>
           <div class="module-read-more-{{display_type}} small no-padding">
-            <a class="module-read-more-{{display_type}}-btn url" href="#" data-toggle="modal" data-target="#module-modal-read-more-{{module.attributes.name }}{{ additionalModalSuffix|default('') }}">{{ 'Read More'|trans({}, 'Admin.Modules.Feature') }}</a>
+            {% if module.attributes.id != "0" %}
```

Comment:
FI, these modules were previously listed in the tab Notifications > to install. By merging all the modules non-installed in the `Catalog` tab, we now have some confusion when looking for their origin.